infectiousRSGF8.go

```diff
@@ -19,10 +19,11 @@ func NewRSGF8Codec() *rsGF8Codec {
 	return &rsGF8Codec{make(map[int]*infectious.FEC)}
 }
 
-// Encode uses uses the infectous RSGF8 codec to encode the provided data
 func (c *rsGF8Codec) Encode(data [][]byte) ([][]byte, error) {
```

Comment:
why are we removing this comment?

Reply:
I added doctrings to the interface rather than the implementation.